makefile.shade

```diff
@@ -20,7 +20,9 @@ functions
         const string UniverseCommitsFileName = "commits-universe";
 
         private static bool Quiet { get; set; }
-
+        private static bool UseGateBranch { get; set; }
+        private const string GateBranch = "gateDev";
```

Comment:
The basic idea I had when writing this is that if you call that target it will look for a specific pre-defined branch, use it if it's available, and if it's not use the normal branch. This way we can have "special" branches for only the affected repos and fail over to dev for the rest.

I share your concern about people leaving the branch laying around, but if I understand your suggestion correctly (that people would add the environment variable to the config while they're trying it then remove it by hand after they're finished) I think it will suffer from basically the same problem, possibly worse since branches are something most people look at more often then CI environment variables.